app/apps/server/bridges/users.ts

```diff
@@ -1,11 +1,24 @@
 import { Random } from 'meteor/random';
 import { UserBridge } from '@rocket.chat/apps-engine/server/bridges/UserBridge';
 import { IUserCreationOptions, IUser } from '@rocket.chat/apps-engine/definition/users';
+import { Meteor } from 'meteor/meteor';
 
-import { setUserAvatar, checkUsernameAvailability, deleteUser, _setStatusTextPromise } from '../../../lib/server/functions';
+import { setUserAvatar, checkUsernameAvailability, deleteUser } from '../../../lib/server/functions';
 import { Users } from '../../../models/server';
 import { Subscriptions, Users as UsersRaw } from '../../../models/server/raw';
 import { AppServerOrchestrator } from '../orchestrator';
+import { UserStatus } from '../../../../definition/UserStatus';
+import { api } from '../../../../server/sdk/api';
+
+const updateUserPresence = function (user: IUser, status = UserStatus.OFFLINE, statusText = ''): void {
+	const { id, username } = user;
+
+	Meteor.call('setUserStatus', status, statusText);
```

Comment:
We have to avoid using `Meteor.call` to avoid depending on the framework as much as possible. Is there an alternative to calling this method, as it wasn't being called before?

Reply:
got it and done